include/deal.II/base/mpi.h

```diff
@@ -1071,6 +1071,25 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
+     * A generalization of the classic `MPI_Bcast` function that accepts
+     * arbitrary data types `T`, as long as `boost::serialize` accepts `T` as an
```

Comment:
```suggestion
     * arbitrary data types `T`, as long as `Utilities::pack` accepts `T` as an
```
This is an implementation detail that we can hide

Reply:
I don't think this detail should be omitted - for data types `T` not supported by `boost::serialize` (or for which we don't provide the appropriate interface in deal.II), the user needs to be in the position to add a `template<typename Archive> void serialize(Archive &, const unsigned int)` function to the class `T` to be wrapped. So I would suggest
```suggestion
     * arbitrary data types `T`, as long as Utilities::pack() (which in turn uses
     * `boost::serialize`, see in Utilities::pack() for details) accepts `T` as an
```
Or is there a simpler way to use it? If anything, I believe the serialization functionality is under-documented in deal.II with some non-obvious interface (do we discuss it in any of the tutorials, by the way?).